src/components/editor/MonacoEditor.tsx

```diff
@@ -187,10 +190,9 @@ function MonacoEditor({
                         theme={
                             theme.palette.mode === 'light' ? 'vs' : 'vs-dark'
                         }
-                        defaultValue={intl.formatMessage({
-                            id: 'common.loading',
-                        })}
-                        value={stringifyJSON(currentCatalog.spec)}
+                        saveViewState={false}
```

Comment:
We do not need the state saved since we have a store and server handling stuff for us.